### ISSUE TITLE
关于byte_infer_perf中的attention_mask和causal（因果掩码）

### ISSUE THREAD

**Issue by zlk0306** [bytedance/ByteMLPerf#152]
在byte_infer_perf中，以byte_infer_perf/llm_perf/backends/GPU/model_impl/modeling_llama3.py为例：

1. LlamaFlashAttention2中的causal功能由is_causal判断

2. LlamaAttention中的causal相关有参数causal_mask，虽然命名为causal_mask，但其定义为causal_mask = attention_mask[:, :, :, : key_states.shape[-2]]，而attention_mask为变长seq padding

3. 同样的，在LlamaSdpaAttention中，causal_mask = ~attention_mask

请问LlamaAttention和LlamaSdpaAttention是否支持因果掩码？
